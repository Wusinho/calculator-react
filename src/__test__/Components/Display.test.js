import { create } from 'react-test-renderer';

import Display from '../../components/display';

it('Should render correctly', () => {
  const elem = create(<Display result="some value" />);
  expect(elem).toMatchSnapshot();
});
