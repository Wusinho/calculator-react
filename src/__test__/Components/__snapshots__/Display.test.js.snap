// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<h1
  className="display-value"
>
  some value
</h1>
`;
